stacks-core/src/uint.rs

```diff
@@ -0,0 +1,891 @@
+use std::{
```

Comment:
Missing license comment (both here and elsewhere)

Reply:
Actually, please drop this file entirely.  There is nothing wrong with `stacks-common/src/util/uint.rs`.